test/integration/suites/batchx509SVID/00-gen-files.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Define directories
+BASE_DIR="/opt/spire/conf"
+SERVER_DIR="${BASE_DIR}/server"
+AGENT_DIR="${BASE_DIR}/agent"
+
+# Create necessary directories
+mkdir -p "${SERVER_DIR}"
+mkdir -p "${AGENT_DIR}"
+
+# Generate Root CA Certificate and Key
+echo "Generating Root CA certificate and key..."
+openssl genrsa -out root-ca.key 2048
```

Comment:
I didn't see other CI generating spire's crypto; I assumed there was some common module doing all this. Do we need to do it bespoke here?

Reply:
we should get a bug open for someone's local env being incompatible with SPIRE's CI setup

otherwise, CI should be consistent. I had a similar issue myself and just made use of this project's existing remote CI whenever a commit is pushed. but rather than diverge patterns for how setup is done, everything should do the same thing and we should figure out separately how to either:
- make that same thing work for your environment
- alter the pattern everywhere, if the one you have here (or some other pattern) can be used more ubiquitously